apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/settings/security/saml.tsx

```
@@ -145,14 +196,47 @@ export function SAML() {
+                  text="Save"
+                  loading={isSubmitting}
+                  disabled={plan !== "enterprise" || !isDirty}
+                  className="w-fit"
+                />
+              </div>
+            </form>
+          ) : (
+            <div className="flex items-center justify-between">
+              <a
+                href="https://dub.co/help/category/saml-sso"
+                target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)